cdap-app-templates/cdap-etl/cdap-data-streams/src/main/java/co/cask/cdap/datastreams/SparkStreamingPipelineDriver.java

```diff
@@ -62,6 +70,11 @@
 
   @Override
   public void run(final JavaSparkExecutionContext sec) throws Exception {
+    String arguments = Joiner.on(", ").withKeyValueSeparator("=").join(sec.getRuntimeArguments());
+    WRAPPERLOGGER.info("<{}> Pipeline '{}' starting with arguments {}",
```

Comment:
The user part in the log message can be confusing. Should explicitly indicate it's user's name like `Pipeline '{}' is started by user <{}> with arguments {}"`